package.json

```diff
@@ -82,6 +82,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "@lifeomic/chromicons": "^0.2.1",
```

Comment:
Asking, just for completeness. Should this be a `peerDependency`? I suppose the only concrete benefit would be that consumers could get new icons released in new minor versions of `chromicons`.

Reply:
I _almost_ put it as a `peerDependency`, but then decided against it for no good reason other than my initial gut feeling 😂 .  I'm happy to add it as a `peerDependency` though!  What are your thoughts here?  Think it'd be better as a `peerDependency`?

Reply:
I suppose it _shouldn't_ matter, since it's just components. multiple copies wouldn't be a problem, and _should_ resolve at runtime in an expected manner, right? (i.e. `chroma` components would get their dependent version of chromicons, and consumer imports should get their (potentially different) version)

Reply:
Exactly my thinking and why I hesitated with `peerDependency`.  Chroma doesn't use a ton of icons internally, but the apps themselves do, so if there is a mismatch in the versioning of Chromicons, it shouldn't be a big deal if Chroma is "behind".

Reply:
If we change our mind in the future, it will be easier to migrate _away_ from peer dependency than _to_ peer dependency. (_to_ peer dependency is a breaking change for consumers)

Reply:
> If we change our mind in the future, it will be easier to migrate _away_ from peer dependency than _to_ peer dependency. (_to_ peer dependency is a breaking change for consumers)

Love it.  Thanks for hashing this out with me 🙇 .  PR incoming!